db/migrate/20230904155318_rename_question_answer_comment_table.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require "migration/table_dropper"
+
+class RenameQuestionAnswerCommentTable < ActiveRecord::Migration[7.0]
+  def up
+    unless table_exists?(:post_voting_comments)
+      Migration::TableDropper.read_only_table(:question_answer_comments)
+      execute <<~SQL
+        UPDATE question_answer_votes
+        SET votable_type = 'PostVotingComment'
+        WHERE votable_type = 'QuestionAnswerComment'
+      SQL
```

Comment:
Since we're already avoid table rename – should this operation be done at the end of this migration and on the new table instead?

Reply:
What do you mean by "Since we already avoid table rename"?

Reply:
I did think of it, but the two table migrations are done in separate files.

So if this first migration for `QuestionAnswerComment` rename to `PostVotingComment` is successful, `question_answer_votes` will is still pending and will be using `PostVotingComment`.

I guess the question is if I should merge the two migrations into one file instead. WDYT?